homework/style.css

```diff
@@ -1,3 +1,176 @@
+* {
```

Comment:
On a mobile phone this looks wrong:

![image](https://user-images.githubusercontent.com/2788771/65972133-ab691380-e469-11e9-8307-df75192c1f1b.png)